<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>776</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../opt_resource.qrc">
    <normaloff>:/images/icon.ico</normaloff>:/images/icon.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QToolButton" name="btnLearnMore">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>160</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Learn More...</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="txtInputAddresses">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>90</y>
      <width>291</width>
      <height>191</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Serif</family>
      <pointsize>9</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QTextEdit" name="txtOptimizedAddresses">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>380</y>
      <width>291</width>
      <height>191</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Serif</family>
      <pointsize>9</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="btnOptimize">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>380</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Optimize</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnOpenMaps">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>520</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Open in Maps</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>50</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Variable Display</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Addresses. Origin address first.</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>110</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Variable Display Semil</family>
      <pointsize>18</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;How to Use?&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCopyLink">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>450</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Copy Maps Link</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>340</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Variable Display</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Optimized results. Approximated.</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblLogo">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>620</y>
      <width>181</width>
      <height>61</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../opt_resource.qrc">:/images/logo.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionRefresh"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionCopy"/>
    <addaction name="actionCut"/>
    <addaction name="actionPaste"/>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHow_to_Use"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionRefresh"/>
   <addaction name="actionCopy"/>
   <addaction name="actionCut"/>
   <addaction name="actionPaste"/>
   <addaction name="actionUndo"/>
   <addaction name="actionRedo"/>
   <addaction name="actionHow_to_Use"/>
   <addaction name="actionAbout"/>
   <addaction name="actionQuit"/>
  </widget>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="../opt_resource.qrc">
     <normaloff>:/images/icons8-quit-50.png</normaloff>:/images/icons8-quit-50.png</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionRefresh">
   <property name="icon">
    <iconset resource="../opt_resource.qrc">
     <normaloff>:/images/icons8-reset-50.png</normaloff>:/images/icons8-reset-50.png</iconset>
   </property>
   <property name="text">
    <string>Refresh</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset resource="../opt_resource.qrc">
     <normaloff>:/images/icons8-copy-paste-64.png</normaloff>:/images/icons8-copy-paste-64.png</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="toolTip">
    <string>Copy highlighted text from the input addresses box</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset resource="../opt_resource.qrc">
     <normaloff>:/images/icons8-cut-50.png</normaloff>:/images/icons8-cut-50.png</iconset>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
   <property name="toolTip">
    <string>Cut highlighted text from the input addresses box</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset resource="../opt_resource.qrc">
     <normaloff>:/images/icons8-paste-50.png</normaloff>:/images/icons8-paste-50.png</iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="toolTip">
    <string>Paste into the input addresses box</string>
   </property>
  </action>
  <action name="actionHow_to_Use">
   <property name="icon">
    <iconset resource="../opt_resource.qrc">
     <normaloff>:/images/icons8-help-48.png</normaloff>:/images/icons8-help-48.png</iconset>
   </property>
   <property name="text">
    <string>How to Use</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="../opt_resource.qrc">
     <normaloff>:/images/icons8-about-50.png</normaloff>:/images/icons8-about-50.png</iconset>
   </property>
   <property name="text">
    <string>About Route Optimizer</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="../opt_resource.qrc">
     <normaloff>:/images/icons8-undo-64.png</normaloff>:/images/icons8-undo-64.png</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="toolTip">
    <string>Undo changes in the input addresses box</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset resource="../opt_resource.qrc">
     <normaloff>:/images/icons8-redo-64.png</normaloff>:/images/icons8-redo-64.png</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="toolTip">
    <string>Redo changes in the input addresses box</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../opt_resource.qrc"/>
 </resources>
 <connections/>
</ui>
